### PR TITLE
Feature/new submission list fields [OSF-6603]

### PR DESCRIPTION
**Pull request by TomHeatwole**
There is still a little bit of work to be done on this after the file upload is ready. I will make a new ticket for that.

### CODE REVIEW

File: app/routes/conference/index.js [hunk 1]
```diff
@@ -7,5 +7,13 @@ export default Ember.Route.extend({
...
             submissions : this.store.findAll('submission')
         });
+    },
+    actions: {
+    	download(uri) {
+		  var link = document.createElement("a");
+		  link.download = '';
+		  link.href = uri;
+		  link.click();
+		},
```
Comment (Ghalko) [inline app/routes/conference/index.js:17]: Fix indents.

File: app/templates/conference/index.hbs [hunk 2]
```diff
@@ -37,14 +36,34 @@
                                 </div>
                             </div>
 
+                            <!-- This is the basic code for a list with images -->
+                            <!--                                   <div id="table" class="row">
```
Comment (Ghalko) [inline app/templates/conference/index.hbs:40]: I am not sure we will ever need a list with images.

File: meetings/submissions/models.py [hunk 3]
```diff
@@ -10,6 +10,7 @@ class Submission(models.Model):
     title = models.CharField(max_length=100)
     contributor = models.ForeignKey(User, related_name='submission_contributor')
     description = models.TextField()
+    # category = models.TextField()
```
Comment (Ghalko) [inline meetings/submissions/models.py]: This should probably be a choice field. And we need to actually put this in.

File: meetings/submissions/serializers.py [hunk 4]
```diff
@@ -41,3 +45,12 @@ def get_can_edit(self, obj):
         request = self.context.get('request')
         user = request.user
         return (user == obj.contributor or user == obj.conference.admin)
+
+    def get_download_count(self, obj):
+        conn = httplib.HTTPConnection("api.osf.io")
```
Comment (Ghalko) [inline meetings/submissions/serializers.py]: Don't hardcode this value. Look at other places that osf is referenced and how it is handled.

File: meetings/submissions/serializers.py [hunk 5]
```diff
@@ -41,3 +45,12 @@ def get_can_edit(self, obj):
...
+        #file_id = Submission.objects.filter() <-- Edit this to find the actual fileid
+        conn.request("HEAD","/v2/files/") #not done needs to be vs/files/{fileid}
+        res = conn.getresponse()
+        # figure out response format and return download count
```
Comment (Ghalko) [inline meetings/submissions/serializers.py]: You should be able to figure out the format for a file response by looking at OSF code or sending requests to the API with postman.

**Comment by Ghalko** [comment]
@TomHeatwole Please close this if it is no longer relevant. Otherwise update it.

**Comment by TomHeatwole** [comment]
No longer relevant. These commits were merged with another branch.

**Pull request closed without merging**